data/attributes.yml

```diff
@@ -13967,9 +13972,9 @@ base_attributes:
   values:
   - hardwood_lumber_grade__firsts_and_seconds_fas
   - hardwood_lumber_grade__no_1_common
-  - hardwood_lumber_grade__no_2_common
```

Comment:
Edge case alpha is not handling well (2a before 2). Should we use custom here?

Reply:
Is this a tune to the regexes moreso? Sounds like we likely always want the generic form first?